cmd/main.go

```diff
@@ -19,6 +20,11 @@ var (
 
 func main() {
 
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatal("Error loading .env file, must be at project root")
+	}
+
```

Comment:
I did do something a little bold here, I moved your .env load into main for 2 reasons:

1. I am fairly certain you were repeatedly reading the `.env` file by calling `GetEnv` over and over in `service`. No reason for that, so I pushed the load right to the beginning so the server just fails instantly if it can't be found
2. All the Go experts I've talked to prefer you do your dependency creation in `main` for a plethora of reasons, mostly because the code becomes more like a library so you can inject your own values if you just want to `go get` one of your subpackages and not need to set environment variables

You should look at libraries like `namsral/flag`, here's a good example from one of my colleagues: https://github.com/akhenakh/kvtiles/blob/master/cmd/kvtilesd/main.go

Reply:
Ayrıca istekleri env koşullarına da bağladınız. Elbette bu, projeyi yeniden oluşturmadan istekleri engellemek için ideal ve mantıklı bir yaklaşımdır, teşekkürler.